openpype/pipeline/load/plugins.py

```diff
@@ -109,16 +163,31 @@ def is_compatible_loader(cls, context):
         ):
             return False
 
-        maj_version, _ = schema.get_schema_version(context["subset"]["schema"])
+        if (
+            cls.extensions is not None
+            and not cls.has_valid_extension(repre_doc)
+        ):
+            return False
+
+        verison_doc = context["version"]
+        subset_doc = context["subset"]
+        maj_version, _ = schema.get_schema_version(subset_doc["schema"])
         if maj_version < 3:
-            families = context["version"]["data"].get("families", [])
+            families = verison_doc["data"].get("families")
         else:
-            families = context["subset"]["data"]["families"]
+            families = context["subset"]["data"].get("families")
+            if families is None:
+                family = context["subset"]["data"].get("family")
+                if family:
+                    families = [family]
```

Comment:
Personally I think the old style code was more readable - but maybe this is fixing something too?
Anyway, let's make sure to at least fix typo `verison` to `version`